crates/bevy_ecs/src/query/access.rs

```diff
@@ -1,14 +1,14 @@
 use crate::storage::SparseSetIndex;
+use bevy_utils::HashSet;
 use fixedbitset::FixedBitSet;
 use std::marker::PhantomData;
 
-/// `Access` keeps track of read and write accesses to values within a collection.
+/// Tracks read and write access to specific elements in a collection.
 ///
-/// This is used for ensuring systems are executed soundly.
-#[derive(Debug, Eq, PartialEq, Clone)]
+/// System initialization and execution uses these to ensure soundness.
+#[derive(Debug, Clone, Eq, PartialEq)]
 pub struct Access<T: SparseSetIndex> {
```

Comment:
These docs should link to `is_compatible`, which is by far the most important public method on this type.

Reply:
Good now?

Reply:
Yep, looks great.